Buildings/Resources/Scripts/Dymola/ThermalZones/Detailed/Examples/ISAT/ForcedConvection.mos

```diff
@@ -0,0 +1,33 @@
+simulateModel("Buildings.ThermalZones.Detailed.Examples.ISAT.ForcedConvection", stopTime=600, method="Cvode", tolerance=1e-06, resultFile="ForcedConvection");
```

Comment:
the `stopTime` is 600s here but in the class `Buildings.ThermalZones.Detailed.Examples.ISAT.ForcedConvection`, it was set to 120 seconds.

Reply:
Revised.